SmthFuzz/ReadMe.txt

```diff
@@ -14,4 +14,4 @@ Edited smoothsw.ini.
 hum.wav Normalized to -5db.
 out.wav Normailized to 0dB and fade out full.
 Edited lock to loop cleanly and have less monophonic-like hum mixed in,
-and additional crackling mixed in.
```

Comment:
There seems to be more changes here than what is described here.
What are the changes?